xlml/utils/xpk.py

```diff
@@ -72,7 +72,7 @@ def run_workload(
             f" --command='{run_cmds}' --device-type={accelerator_type}"
             f" --{multi_keyword}={num_slices} --docker-image={docker_image}"
             f" --project={cluster_project} --zone={zone}"
-            f" --env {metric_config.SshEnvVars.GCS_OUTPUT.name}={gcs_path}"
+            f" --env {metric_config.SshEnvVars.BASE_OUTPUT_PATH.name}={gcs_path}"
```

Comment:
Other tests currently depend on `GCS_OUTPUT` being set. Could we prefix an `export BASE_OUTPUT_PATH=$GCS_OUTPUT` to the train command instead?

Reply:
Perfect - done - thank you!